pkgs/development/python-modules/pivy/default.nix

```diff
@@ -16,8 +16,12 @@ buildPythonPackage rec {
     sha256 = "18n14ha2d3j3ghg2f2aqnf2mks94nn7ma9ii7vkiwcay93zm82cf";
   };
 
+  nativeBuildInputs = with pkgs; [
+    swig1 coin3d soqt
```

Comment:
You're sure these should *only* be native? Did you actually try using pivy?

Reply:
No, not sure.  I tested compilation and running FreeCAD.  I don't know how to test, from within FreeCAD, that the coin3d library is called correctly.  Should I add it to `buildInputs` too, to be on the safe side?

Reply:
My intuition is that both, coin3d and soqt, don't belong into `nativeBuildInputs` but `buildInputs`. If that works, I guess we can leave it as is.
However, we should eventually bring pivy up to date and maybe try running the tests as in https://github.com/FreeCAD/pivy/blob/0.6.4/.travis.yml#L20.

Reply:
Well, they were there, until b4acd9772975d549f491d48debb679cf4ec78133 specifically set `strictDeps` to true, but states that compile/runtime deps belong to `buildInputs`.  These two things seem to be opposed to each other, though, since this has the effect of `CFLAGS` not 
including the `buildInputs` dependencies anymore.  Equally probable that I misunderstood the whole thing, though.

TLDR: no idea, really.

Note that there is another WIP PR (see description) which deals with bringing pivy and everything else up-to-date.

Reply:
Afaik, `swig` is only used at build time whereas the other two are runtime dependencies. Hence `coin3d` and `soqt` should definitely be in `buildInputs`. After looking at this further I think it is required to also put them in `nativeBuildInputs` because `coin-config` `soqt-config` are called at build time.

Reply:
Done